### DISCUSSION TITLE
Sample Projects for use in Browser

### DISCUSSION THREAD

**Discussion by johnmbailey** [DomCR/ACadSharp#413] [Q&A]
I currently utilize 2 separate Libraries to allow me to create a drawing markup with labels and links to DB data using a Windows Desktop program (woute), and then another library to open the marked up file in a browser to view the DB data associated with the markup / labels.    We do not need to add significant details to an existing drawing, but simply navigate within the drawing and then add bubble tags with linked data.   We also might simply edit colors or highlight elements of the drawings being loaded.  Is there any sample code that shows current capabilities for a web application or Windows desktop program?

**Comment by DomCR** [comment]
Hi @johnmbailey 

ACadSharp does not have an inherit UI implementation, if you want to use it in a UI you will need to integrate it in another project.
